pom.xml

```diff
@@ -1,23 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>nl.knaw.dans.api</groupId>
+   <parent>
```

Comment:
Added dans-prototype as parent, to inherit installation package building. I will add dans-prototype later to Github, so we don't depend on things that have not been published.